<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1259</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Coline - 201513696</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPushButton" name="btnAnalizar">
       <property name="text">
        <string>Compilar 3D</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnAceptar">
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnDepurar">
       <property name="text">
        <string>Depurar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnContinuar">
       <property name="text">
        <string>Automatico</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSiguiente">
       <property name="text">
        <string>Siguiente Punto</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSaltar">
       <property name="text">
        <string>Compilar 3D</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnAutomatico">
       <property name="text">
        <string>--</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnExportar">
       <property name="text">
        <string>--</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnResetDbg">
       <property name="text">
        <string> Reset Debg</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnAux2">
       <property name="text">
        <string>Punto Interrupcion</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtPuntos"/>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QTabWidget" name="tabColine">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
       <widget class="QTabWidget" name="tab3d">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="widget">
         <attribute name="title">
          <string>C3D</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTextEdit" name="txtSalida">
            <property name="font">
             <font>
              <family>Nimbus Mono L</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="tabStopWidth">
             <number>30</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Modulos</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QTextEdit" name="txtEntrada">
            <property name="font">
             <font>
              <family>Nimbus Mono L</family>
             </font>
            </property>
            <property name="tabStopWidth">
             <number>32</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tbSalida">
        <attribute name="title">
         <string>Salida</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPlainTextEdit" name="txtConsolaSalida2">
           <property name="palette">
            <palette>
             <active>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Button">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>64</red>
                 <green>66</green>
                 <blue>68</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Light">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>96</red>
                 <green>99</green>
                 <blue>102</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Midlight">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>80</red>
                 <green>82</green>
                 <blue>85</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Dark">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>32</red>
                 <green>33</green>
                 <blue>34</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Mid">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>42</red>
                 <green>44</green>
                 <blue>45</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Text">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="BrightText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>64</red>
                 <green>66</green>
                 <blue>68</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Shadow">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="AlternateBase">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>32</red>
                 <green>33</green>
                 <blue>34</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ToolTipBase">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>220</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ToolTipText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Button">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>64</red>
                 <green>66</green>
                 <blue>68</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Light">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>96</red>
                 <green>99</green>
                 <blue>102</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Midlight">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>80</red>
                 <green>82</green>
                 <blue>85</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Dark">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>32</red>
                 <green>33</green>
                 <blue>34</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Mid">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>42</red>
                 <green>44</green>
                 <blue>45</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Text">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="BrightText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>64</red>
                 <green>66</green>
                 <blue>68</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Shadow">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="AlternateBase">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>32</red>
                 <green>33</green>
                 <blue>34</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ToolTipBase">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>220</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ToolTipText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>32</red>
                 <green>33</green>
                 <blue>34</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Button">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>64</red>
                 <green>66</green>
                 <blue>68</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Light">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>96</red>
                 <green>99</green>
                 <blue>102</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Midlight">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>80</red>
                 <green>82</green>
                 <blue>85</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Dark">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>32</red>
                 <green>33</green>
                 <blue>34</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Mid">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>42</red>
                 <green>44</green>
                 <blue>45</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Text">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>32</red>
                 <green>33</green>
                 <blue>34</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="BrightText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>32</red>
                 <green>33</green>
                 <blue>34</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>64</red>
                 <green>66</green>
                 <blue>68</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>64</red>
                 <green>66</green>
                 <blue>68</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Shadow">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="AlternateBase">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>64</red>
                 <green>66</green>
                 <blue>68</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ToolTipBase">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>220</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ToolTipText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="font">
            <font>
             <family>Nimbus Mono L</family>
            </font>
           </property>
           <property name="plainText">
            <string>&gt;&gt;</string>
           </property>
           <property name="tabStopWidth">
            <number>30</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tbErrores">
        <attribute name="title">
         <string>Errores</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QTableWidget" name="tablaErrors">
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>200</number>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Linea</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Columna</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Tipo</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Descripcion</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Tabla de Simbolos</string>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1259</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="menuNuevo"/>
   </widget>
   <addaction name="menuFIle"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="menuNuevo">
   <property name="text">
    <string>Nueva Pestaña</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
